db/migrate/20210309064924_remove_profile_columns_from_user.rb

```diff
@@ -0,0 +1,33 @@
+class RemoveProfileColumnsFromUser < ActiveRecord::Migration[6.0]
```

Comment:
YOLO 🤣

Reply:
Haha, luckily we removed all their usages in the previous PR which didn't blow up too many things. Well, at least not things we noticed 🤣

Reply:
this is the exciting culmination of a long process. I'm so very happy to see all these `alter table, drop column` commands.